lib/logging/CBasicLogConfigurator.h

```diff
@@ -33,7 +33,7 @@ class DLL_LINKAGE CBasicLogConfigurator
 	void configureDefault();
 
 	/// Removes all targets from the global logger.
-	void deconfigure();
+	static void deconfigure();
```

Comment:
HAve not seen where it is used.

Reply:
removed this function

Reply:
it seems it needed on some platforms for cancel logger init. Restored.